docs/content/how-to/aws/migrate-hosted-cluster.md

```diff
@@ -0,0 +1,990 @@
+---
+title: Migrating Hosted Cluster among the same AWS Region
+---
+
+# Migrating Hosted Cluster among the same AWS Region
```

Comment:
Is there anything regarding storage we want to mention?

Reply:
Sure and I also added some comments about the DNS part